bib/pubs.bib

```diff
@@ -46,6 +46,15 @@ @inproceedings{kamahori2025liteasr
       url={https://arxiv.org/abs/2502.20583}, 
 }
 
+@inproceedings{zhu2025tactic,
```

Comment:
To make it work, you have to manually trigger `make` which generates https://github.com/uwsampl/uwsampl.github.io/blob/main/_includes/pubs.html.